syntax-highlighting-code-block.php

```diff
@@ -728,7 +731,8 @@ function customize_register( $wp_customize ) {
 function override_highlighted_line_background_color_post_value( WP_Customize_Manager $wp_customize ) {
 	$highlighted_line_background_color_setting = $wp_customize->get_setting( 'syntax_highlighting[highlighted_line_background_color]' );
 	if ( $highlighted_line_background_color_setting && ! $highlighted_line_background_color_setting->post_value() ) {
-		$highlighted_line_background_color_setting->default = get_default_line_background_color( get_plugin_option( 'theme_name' ) ); // This has no effect.
+		// This is the default value used when highlighting lines on first load in Customizer.
+		$highlighted_line_background_color_setting->default = get_plugin_option( 'highlighted_line_background_color' );
```

Comment:
I can't seem to cause this code to execute as the comment indicates. At least, it runs only in the Customizer preview window not in the `customize.php` window where the control is. Therefore the change here doesn't seem to make any change. And the return values are the same:

`get_default_line_background_color( get_plugin_option( 'theme_name' ) )` ➡️  `#ddf6ff`
`get_plugin_option( 'highlighted_line_background_color' )` ➡️  `#ddf6ff`

I guess the change here is rather just a refactor since the latter code added is the same as the former code? If so, should the comment change be reverted?

Reply:
Actually, I can see a reason for going back to `get_default_line_background_color( get_plugin_option( 'theme_name' ) )`, with some modification. What if someone is using the `syntax_highlighting_code_block_style` filter? The default style should reflect what that theme is, not what the option is, since we aren't using the option at all, as the filtered value is overriding it. I've tried that in a3078c7.